int64 actor_id
float64 speed
float64 acceleration
geometry_msgs/Pose2D location_global
# FrenetMsg location_frenet
float64 length
float64 width
